### PR TITLE
feat/docs/fix: changed the colour of the scroll to top button

### PR DESCRIPTION
**Pull request by Bhanuteja01**
## Related Issue
#715

## Description
Changed the colour of the scroll to top button

## Screenshots
Screenshot (6)

Screenshot (5)

### CODE REVIEW

**Comment by ydv-ankit** [comment]
there are some issues. should correct them

File: package-lock.json
Comment (priyankarpal) [inline package-lock.json:1]: remove this file. Please check docs we are suing `pnpm` so use `pnpm` instead of `npm`

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by priyankarpal** [comment]
also bg not showing

**Comment by Bhanuteja01** [comment]
> also bg not showing

what is reason it not showing now

**Pull request closed without merging**